Exiled.Events/Patches/Generic/TutorialHandcuffImmunity.cs

```diff
@@ -0,0 +1,54 @@
+// -----------------------------------------------------------------------
+// <copyright file="PlayerCuffedPatch.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Generic
+{
+    using System.Collections.Generic;
+    using System.Reflection.Emit;
+
+    using Exiled.API.Features;
+    using Exiled.API.Features.Pools;
+    using HarmonyLib;
+    using InventorySystem.Disarming;
+    using PlayerRoles;
+
+    using static HarmonyLib.AccessTools;
+
+    /// <summary>
+    /// Patches <see cref="DisarmingHandlers.ServerProcessDisarmMessage"/>.
+    /// </summary>
+    [HarmonyPatch(typeof(DisarmingHandlers), nameof(DisarmingHandlers.ServerProcessDisarmMessage))]
+    internal static class TutorialHandcuffImmunity
+    {
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
+        {
+            List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Pool.Get(instructions);
+
+            Label continueLabel = generator.DefineLabel();
+            Label skipLabel = generator.DefineLabel();
+
+            newInstructions.InsertRange(0, new CodeInstruction[]

```

Comment:
For styling & consistency, Use `new[]` instead of `new CodeInstruction[]`.

```suggestion
            newInstructions.InsertRange(0, new[]
```